app/javascript/app/components/tab-switcher/tab-switcher-styles.scss

```diff
@@ -24,6 +25,11 @@
 
 .switch {
   height: 40px;
+  margin-bottom: $gutter-padding;
+
+  @media #{$tablet-landscape} {
+    margin-bottom: 0;
```

Comment:
The `margin-bottom` is applied before the `. toolbarActions` wrap under the `.switch` creating a layout break. It happens between `900px` and `840px`.